### PR TITLE
Project - Marina Sánchez

### PR DESCRIPTION
**Pull request by MarinaSM95**
I have to fix some details (image dimensions and make the last image visible). Plus, in the .html version, at the heading "Storing the new data frame" the format expands (I don't understand why). 

Any comment is more than welcome!

### CODE REVIEW

File: _projects/2022/100462872/100462872.Rmd [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
+
+
```
Comment (Enchufa2) [inline _projects/2022/100462872/100462872.Rmd]: The Rmd is empty! Maybe you committed the wrong one?

Comment (MarinaSM95) [reply _projects/2022/100462872/100462872.Rmd]: Yes! I think is ok now (I committed and pushed the correct .Rmd and .html)

**Comment by Enchufa2** [comment]
Splendid work! After some fixes with the figure widths and so on (see [the changes](https://github.com/IBiDat/dataviz/pull/27/commits/8ff942554e830c90195466b73accb0c23060d381#diff-382ab0fa5dac92cefc98fcf599c51c6329f962a4d65f54f9c174c956ca384255) clicking "load diff"), it looks great to me! Merging...

**Comment by Enchufa2** [comment]
Online: https://csslab.uc3m.es/dataviz/projects/2022/100462872/

**Pull request merged**